pkg/nsx/services/subnet/subnet.go

```diff
@@ -38,6 +39,17 @@ type SubnetService struct {
 	common.Service
 	SubnetStore *SubnetStore
 	builder     *common.PolicyTreeBuilder[*model.VpcSubnet]
+	// SharedSubnetsMap is a map of namespaced names to associated resources for shared subnets that need polling
+	SharedSubnetsMap map[types.NamespacedName]string
+	// mutex to protect the SharedSubnetsMap map
+	SharedSubnetsMutex sync.RWMutex
+	// NSXSubnetCache is a cache of associatedResource -> nsxSubnet and statusList mapping
+	NSXSubnetCache map[string]struct {
```

Comment:
Rename or update to comment to state the cache is only for precreated subnet?

Reply:
added